src/Web/ClientApp/src/app/core/api/v1/model/appDto.ts

```diff
@@ -9,14 +9,14 @@
  * https://openapi-generator.tech
  * Do not edit the class manually.
  */
-import { ChannelSummaryDto } from './channelSummaryDto';
+import { ApplicationChannelSummary } from './applicationChannelSummary';
 
 
 export interface AppDto { 
     id: string;
     name: string;
     storageId: string;
     description: string;
-    channels: Array<ChannelSummaryDto>;
+    channels: Array<ApplicationChannelSummary>;
```

Comment:
What is the difference between an "ApplicationChannelSummary" and a "ChannelSummaryDto"? Aren't they the same thing? Why do we have three ways to present the same data?